oaknational/Oak-Web-Application
# Added swimming to curriculum downloads (docx)

## Description
> [!IMPORTANT]  
> This is branched off of https://github.com/oaknational/Oak-Web-Application/pull/2816 so that PR must land first

Added swimming items into unit sequence pages

<img width="601" alt="Screenshot 2024-09-25 at 14 52 22" src="https://github.com/user-attachments/assets/1176c5c6-136c-42b5-8f5e-f69ef50e1cff">

<img width="601" alt="Screenshot 2024-09-25 at 14 52 34" src="https://github.com/user-attachments/assets/6662724a-5b53-4104-8e7d-8ca772557ad8">

## Issue(s)

Fixes

 - `CUR-847`
 - `CUR-846`
 - `CUR-845`

## How to test
Cycle 1 

1. Go to https://deploy-preview-2820--oak-web-application.netlify.thenational.academy
2. Check for regressions

Cycle 2

1. Check out locally
2. Enable `ENABLE_CYCLE_2`/`SWIMMING_HACK`
3. Test mentioned tickets above
